### PR TITLE
`f-27-automate`: Please R CMD Check

### PR DESCRIPTION
**Pull request by pat-s**
To get green builds for this branch.

### CODE REVIEW

File: R/auto.R [hunk 1]
```diff
@@ -416,7 +416,13 @@ create_pull_request <- function(release_branch, main_branch, remote_name, force)
     config_url <- glue("branch.{release_branch}.pr-url")
     rlang::exec(git2r::config, !!config_url := NULL)
 
-    create <- is.null(usethis:::pr_url())
+    pr_list <- gh::gh("/repos/cynkra/fledge/pulls",
```
Comment (krlmlr) [inline R/auto.R:419]: Do we need to parameterize this?

File: R/auto.R [hunk 2]
```diff
@@ -416,7 +416,13 @@ create_pull_request <- function(release_branch, main_branch, remote_name, force)
...
+      "head" = get_branch_name(), "sort" = "created"
+    )
+    create <- !any(vapply(pr_list, function(x) x$head$ref,
```
Comment (krlmlr) [inline R/auto.R:422]: Please extract a variable from the `vapply()` call.

File: R/gh-helpers.R [hunk 3]
```diff
@@ -9,11 +9,11 @@ get_remote_url <- function(path, remote) {
   r <- git2r::repository(path, discover = TRUE)
   remote_names <- git2r::remotes(r)
   if (!length(remote_names)) {
-    stopc("Failed to lookup git remotes")
+    stop("Failed to lookup git remotes")
```
Comment (krlmlr) [inline R/gh-helpers.R:12]: `stopc()` was a helper function, gone now?

File: vignettes/out/demo.md [hunk 4]
```diff
@@ -35,46 +35,46 @@ for development. The output shows the details of the package created.
...
-    ## [32m✔[39m Creating [34m'/tmp/RtmpGalley/fledge/SuperFrob/'[39m
-    ## [32m✔[39m Setting active project to [34m'/tmp/RtmpGalley/fledge/SuperFrob'[39m
-    ## [32m✔[39m Creating [34m'R/'[39m
-    ## [32m✔[39m Writing [34m'DESCRIPTION'[39m
-    ## [34mPackage[39m: SuperFrob
-    ## [34mTitle[39m: What the Package Does (One Line, Title Case)
-    ## [34mVersion[39m: 0.0.0.9000
-    ## [34mDate[39m: 2020-12-12
-    ## [34mAuthors@R[39m (parsed):
+    ## [32m✔[39m Creating [34m'/tmp/RtmpGalley/fledge/SuperFrob/'[39m
```
Comment (krlmlr) [inline vignettes/out/demo.md:38]: Where do these whitespace changes come from?

File: DESCRIPTION [hunk 5]
```diff
@@ -28,6 +28,7 @@ Imports:
     foghorn,
     gert (>= 1.0.0),
     gh,
+    git2r,
```
Comment (krlmlr) [inline DESCRIPTION:31]: We won't need this after #85.

**Review by krlmlr — COMMENTED** [review]
Thanks for working on this.

File: R/auto.R (hunk 1 above)
Comment (pat-s) [reply R/auto.R:419]: Ah ofc, my bad

File: R/auto.R (hunk 2 above)
Comment (pat-s) [reply R/auto.R:422]: Done

File: R/gh-helpers.R (hunk 3 above)
Comment (pat-s) [reply R/gh-helpers.R:12]: It seems so, this is really just a quick fix for R CMD Check and imo should be added back in `f-27-automate` then?

File: vignettes/out/demo.md (hunk 4 above)
Comment (pat-s) [reply vignettes/out/demo.md:38]: No idea atm.

File: DESCRIPTION (hunk 5 above)
Comment (pat-s) [reply DESCRIPTION:31]: Yes, but the idea of this PR was to be merged before #85 to have a valid CI check for future work on `f-27-automate`

**Comment by krlmlr** [comment]
R CMD check is fine in #85, let's see later if it's still necessary.

**Pull request closed without merging**